apps/meteor/client/views/home/cards/CustomContentCard.tsx

```diff
@@ -40,6 +40,18 @@ const CustomContentCard = (): ReactElement | null => {
 	const isEnterprise = data?.isEnterprise;
 	const willNotShowCustomContent = isCustomContentBodyEmpty || !isCustomContentVisible;
 
+	const userVisibilityTooltipText = !isCustomContentVisible ? t('Now_Its_Visible_Only_For_Admins') : t('Now_Its_Visible_For_Everyone');
+
+	let customContentOnlyTooltipText;
+
+	if (willNotShowCustomContent) {
+		customContentOnlyTooltipText = t('Action_Available_After_Custom_Content_Added_And_Visible');
+	} else if (isCustomContentOnly) {
+		customContentOnlyTooltipText = t('It_Will_Show_All_Other_Content_Blocks_In_The_Homepage');
+	} else {
+		customContentOnlyTooltipText = t('It_Will_Hide_All_Other_Content_Blocks_In_The_Homepage');
+	}
```

Comment:
start with  the else value 
```suggestion
	let customContentOnlyTooltipText = customContentOnlyTooltipText = t('It_Will_Hide_All_Other_Content_Blocks_In_The_Homepage');

	if (willNotShowCustomContent) {
		customContentOnlyTooltipText = t('Action_Available_After_Custom_Content_Added_And_Visible');
	} else if (isCustomContentOnly) {
		customContentOnlyTooltipText = t('It_Will_Show_All_Other_Content_Blocks_In_The_Homepage');
	}
```